c2corg_api/search/search_filters.py

```diff
@@ -48,6 +50,9 @@ def build_query(url_params, meta_params, doc_type):
         if doc_type == OUTING_TYPE:
             search = search.sort(
                 {'date_end': {'order': 'desc'}}, {'id': {'order': 'desc'}})
+        elif doc_type == IMAGE_TYPE:
+            search = search.sort(
+                {'date_time': {'order': 'desc'}}, {'id': {'order': 'desc'}})
```

Comment:
desc or asc?
The oldest first because it makes more sense to show the images in the order they have been taken.